### PR TITLE
Update `Presto` to `Trino` in PostgreSql

### PR DESCRIPTION
**Pull request by polaris6**
## Description
Update `Presto` to `Trino` in PostgreSql

## Documentation

(x) No documentation is needed.

## Release notes
(x) No release notes entries required.

### CODE REVIEW

**Comment by polaris6** [comment]
Hi @wendigo @leveyja, could you please help me to review this PR when you have time? Thank you very much!

File: plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraMetadata.java [hunk 1]
```diff
@@ -72,6 +72,7 @@
         implements ConnectorMetadata
 {
     public static final String PRESTO_COMMENT_METADATA = "Presto Metadata:";
+    public static final String TRINO_COMMENT_METADATA = "Trino Metadata:";
```
Comment (ebyhr) [inline plugin/trino-cassandra/src/main/java/io/trino/plugin/cassandra/CassandraMetadata.java:75]: Please revert this change.

File: service/trino-verifier/src/main/java/io/trino/verifier/TrinoVerifier.java [hunk 2]
```diff
@@ -15,9 +15,9 @@
 
 import picocli.CommandLine;
 
-public final class PrestoVerifier
+public final class TrinoVerifier
```
Comment (ebyhr) [inline service/trino-verifier/src/main/java/io/trino/verifier/TrinoVerifier.java:18]: Could you revert and send another PR to change this?

File: plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/jmx/JdbcClientStats.java [hunk 3]
```diff
@@ -293,9 +293,9 @@ public JdbcApiStats getSchemaExists()
 
     @Managed
     @Nested
-    public JdbcApiStats getToPrestoType()
+    public JdbcApiStats getToTrinoType()
```
Comment (ebyhr) [inline plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/jmx/JdbcClientStats.java:296]: Could you revert and send another PR to change this?

**Review by ebyhr — COMMENTED** [review]
Could you separate PR for changes it has user impact?

**Review by ebyhr — APPROVED** [review]
